src/main/scala/mesosphere/marathon/core/task/bus/TaskStatusObservables.scala

```diff
@@ -1,13 +1,14 @@
 package mesosphere.marathon.core.task.bus
 
-import mesosphere.marathon.core.task.Task
-import mesosphere.marathon.core.task.bus.TaskStatusObservables.TaskStatusUpdate
-import mesosphere.marathon.state.{ PathId, Timestamp }
+import mesosphere.marathon.core.task.bus.TaskStatusObservables.TaskUpdate
+import mesosphere.marathon.core.task.{ Task, TaskStateChange, TaskStateOp }
+import mesosphere.marathon.state.PathId
 import rx.lang.scala.Observable
 
 object TaskStatusObservables {
-  case class TaskStatusUpdate(timestamp: Timestamp, taskId: Task.Id, status: MarathonTaskStatus) {
-    def appId: PathId = taskId.appId
+  case class TaskUpdate(stateOp: TaskStateOp, stateChange: TaskStateChange) {
```

Comment:
I would prefer `TaskChanged`. 

* `TaskUpdate` is very close to `TaskStateOp.Update` and not restricted to that.
* The past tense suggests that it is a notification of something that happened already.